src/model/notebook/finetune_llama2/OpenThaiGPT_FineTune_Llama.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
I will not go through reviewing this notebook code but please make sure that

1. Please add readme for this notebook (just quick and simple readme is enough)
2. If this notebook is Colab exported
    2.1.  `pip install ...` should be presented inside
    2.2.  please add "Open in Colab" link in readme. example: https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb

3. If this notebook is intended to run on Lanta
3.1 please make sure this code can be run by just `pip install -r ./src/model` if not please describe how to install depedencies